Selenium_Basics/Selenium_Basics/Selenium_Basics_2/EpamPageInteractionTests.cs

```diff
@@ -0,0 +1,102 @@
+﻿using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+using System.Collections.Specialized;
+
+namespace Selenium_Basics.Selenium_Basics_2
+{
+    public class EpamPageInteractionTests
+    {
+        private Actions _action;
+        private IWebDriver _driver;
+        private const string _searchIconButtonLocator = "//span [contains(@class, 'search-icon')]";
+        private const string _searchFormFieldLocator = "//*[@id = 'new_form_search']";
+        private const string _findButtonLocator = "//span [contains (@class, 'bth')]";
+
+
+        [SetUp]
+        public void Setup()
+        {
+            _driver = new ChromeDriver();
+            _action = new Actions(_driver);
+            _driver.Manage().Window.Maximize();
+            _driver.Navigate().GoToUrl("https://www.epam.com/");
+        }
+
+        [Test]
+        public void CheckAvailableCountriesIsPresentOnJobListingsPage()
+        {
+            _driver.FindElement(By.XPath("//*[contains(@class, 'item-link') and @href = '/careers']")).Click();
+            _driver.FindElement(By.XPath("//*[@class = 'arrow']//ancestor::*[contains (@href, 'job-listings') and @tabindex = '0']")).Click();
+            _action.ScrollToElement(_driver.FindElement(By.XPath("//*[contains (@href, 'net-developer-middle')]"))).Build().Perform();
+
+            var findCountries = _driver.FindElements(By.XPath("//*[contains (@class, 'js-tabs-title')]"));
+            var actualCountries = findCountries.Select(f => f.GetDomProperty("innerText").ToList());
+
+            List<string> expectedCountries = new() { "AMERICAS", "EMEA", "APAC" };
+
+            Assert.That(actualCountries, Is.EqualTo(expectedCountries), "Countries are not equal");
```

Comment:
please try to use CollectionAssert :) 
You will get all 2 lists in console in case of error :)